test-app/src/LazyApp.ts

```diff
@@ -1,12 +1,9 @@
-import WidgetBase from '@dojo/framework/widget-core/WidgetBase';
-import { v, w } from '@dojo/framework/widget-core/d';
+import WidgetBase from '@dojo/framework/core/WidgetBase';
+import { v, w } from '@dojo/framework/core/vdom';
 import Outlet from '@dojo/framework/routing/Outlet';
 import LazyWidget from './LazyWidget';
 import StaticAssetWidget from './StaticAssetWidget';
 import RoutedWidget from './RoutedWidget';
-import theme from '@dojo/themes/dojo';
```

Comment:
@matt-gadd Had to remove @dojo/themes and @dojo/widgets until it is upgraded with support for 6.0.0-alpha.2